.examples/docker-compose/with-traefik2-postgresql-redis-multi-network/README.md

```diff
@@ -0,0 +1,23 @@
+## Trafik Multi Network Deployment
```

Comment:
what does "Multi Network" mean in this context, and why would I choose it over the single-network?

Reply:
Example shows 2 networks - one is "traefik" network which used to expose Nextcloud app to reverse proxy and the second one is "Nextcloud" network used for communication between Nextcloud app and DB.

"Multi network" is best practice in docker environment. Basically it means containers belonging to the same application connect to each other using isolated internal network (which is not accessible from outside). This increase security as only container within this network can talk to each other. it's a docker implementation of separated network VLAN.